### ISSUE TITLE
[Suggestion] The ability to see how much liquid there’s in an ender tank

### ISSUE THREAD

**Issue by ghost** [Chicken-Bones/EnderStorage#85]
Some text when looking at the tank and/or a GUI would be nice.

**Comment by Ecu** [comment]
I would say no.  Interacting with things in world is a primary feature of Ender Storage and I feel like popups like this would go against that design.

**Comment by ghost** [comment]
@Ecu Then, like I said, some simple text that shows how many millibuckets or buckets that’s in a tank would suffice. Currently, it’s impossible to see how much liquid is in a tank, which is annoying.

**Comment by Ecu** [comment]
Tough?  This is not a feature I would like to see in my mod.  I feel it is something that is better represented by one of the other tooltip mods, if they want to offer such support.

I know I'm a bit hardline on this and you probably see is as a harmless addition, however, I don't really feel it adds to the gameplay.  It also goes against the idea of having information being handled in world, which is a staple of the mod.

**Comment by ghost** [comment]
Then add it to the config.

Most people want to know how much liquids are in their tanks.

**Comment by Ecu** [comment]
No.  I don't agree with the idea of adding everything that someone might want as a config option.  When I consider features to be added to the config, a main component is whether or not they will offer solid gameplay elements.  This suggestion doesn't offer anything that would really change gameplay, I'm sorry.

**Comment by ghost** [comment]
Why are you acting as though you’re the developer?

**Comment by Ecu** [comment]
Because I am?  I'm the original creator of Ender Storage.  You can see such mentioned here https://minecraft.curseforge.com/projects/ender-storage-1-8 on the latest release.

Additionally, the current maintainer @covers1624 has stated that my viewpoints on features will be respected.  Something to note, this repo is out of date.  It may be better for you to post suggestions on the updated repo over here -> https://github.com/TheCBProject/EnderStorage or at the dedicated feedback repo over here -> https://github.com/TheCBProject/Feedback

You can also hop onto the IRC.  I'm sure others there will confirm my statements here.

**Comment by ghost** [comment]
Well, then. I thought that chicken_bones was the only developer. It’d be nice if GitHub made this a little clearer.

**Comment by Ecu** [comment]
Chicken-Bones assisted me in creating Ender Storage very early on, improving what I had done.  I've always been in the credits screen for the mod, but few people pay attention to it.  I also haven't taken an active role in development of the mod after Chicken-Bones took over that aspect.  Instead, I focus on design regarding the mod, doing some of the art and keeping the features clean.

I also look out to make sure the mod has an active maintainer.  @covers1624 has been doing a great job as the current maintainer.

Edit:  @PapiDimmi you could actually just read the LICENSE.txt here even to see that I am indeed a developer.
